src/uct/tcp/tcp_iface.c

```diff
@@ -70,7 +70,7 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
    "option usually provides better performance",
    ucs_offsetof(uct_tcp_iface_config_t, sockopt_nodelay), UCS_CONFIG_TYPE_BOOL},
 
-  UCT_SEND_RECV_BUF_FIELDS("IFACE_", auto, ucs_offsetof(uct_tcp_iface_config_t, sockopt)),
+  UCT_TCP_SEND_RECV_BUF_FIELDS("IFACE_", auto, ucs_offsetof(uct_tcp_iface_config_t, sockopt)),
```

Comment:
why do we need IFACE_ prefix here?

Reply:
so that there won't be 2 env params with the same name on the same component - https://github.com/openucx/ucx/pull/5073#discussion_r428479736

Reply:
but sockcm has TCP_CM_ prefix so it's not really the same
we need to fix gtest somehow to modify the right params  (or all params which match, if that works)

Reply:
but wouldn't it make sense to have IFACE_ for tcp tl and CM_ for tcp_sockcm?

Reply:
there should already be a unique prefix for the whole variable, no need for another one in the middle of the variable name